fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTray.kt

```diff
@@ -48,10 +58,12 @@ fun TabsTray(
             .fillMaxSize()
             .background(FirefoxTheme.colors.layer1),
     ) {
-        TabsTrayBanner(
-            isInMultiSelectMode = multiselectMode is TabsTrayState.Mode.Select,
-            onTabPageIndicatorClicked = animateScrollToPage,
-        )
+        Box(modifier = Modifier.nestedScroll(rememberNestedScrollInteropConnection())) {
```

Comment:
It's to communicate with the `BottomSheetBehavior` that wraps the `ComposeView` to give the UI the feel of a bottom sheet without explicitly implementing a `BottomSheet` View or Composable. (see: [`TabSheetBehaviorManager`](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabSheetBehaviorManager.kt) and [`TabsTrayFragment`](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFragment.kt#249-268)) This hook allows *just* the banner to be the thing that can be scrolled to expand/collapse the Tabs Tray.

Otherwise, we have something like this, where the tabs tray will collapse when swiping down on the tab list:

https://user-images.githubusercontent.com/87384386/224833665-990d26b4-2a73-4ddf-b04a-99cc50470d5a.mov

It took a couple hours of trial and error to figure out where that `Modifier.nestedscroll` needed to live.